var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -156,6 +157,7 @@ def cmake_args(self):
                     '-DSEACASProj_ENABLE_SEACASAprepro:BOOL=ON',
                     '-DSEACASProj_ENABLE_SEACASAprepro_lib:BOOL=ON',
                     '-DSEACASProj_ENABLE_SEACASConjoin:BOOL=ON',
+                    '-DSEACASProj_ENABLE_SEACASCpup:BOOL=ON',
```

Comment:
Notice that there's helper functions in spack:

```suggestion
                    self.define('SEACASProj_ENABLE_SEACASCpup', 'ON'),
```

which may be a bit more readable -- not a requirement for this PR, but just so you know

Reply:
I will save that for a refactor where I can change all of the lines to follow the suggested form.  Thanks for the suggestion.